olive/systems/system_config.py

```diff
@@ -72,18 +69,29 @@ def _validate_python_environment_path(cls, v):
         return v
 
 
+class PythonEnvironmentTargetUserConfig(CommonPythonEnvTargetUserConfig):
+    olive_managed_env: bool = False  # if True, the environment will be created and managed by Olive
+    requirements_file: Union[Path, str] = None  # path to the requirements.txt file
+
+
+class ORTEnvironmentTargetUserConfig(CommonPythonEnvTargetUserConfig):
```

Comment:
nit: ORTEnvironmentTargetUserConfig = CommonPythonEnvTargetUserConfig?

Reply:
Will keep it as it is for now to keep the inheritance structure as discussed offline.